### ISSUE TITLE
Also check the firefox-trunk folder for Firefox nightly.

### ISSUE THREAD

**Issue by PoorPocketsMcNewHold** [manilarome/blurredfox#36]
![](https://user-images.githubusercontent.com/13170204/92476678-a404cb00-f1df-11ea-8796-bb36e53686f9.png)

Quick explanation as I also found this pretty stupid.

On Debian based distro, Firefox distribute Firefox-nightly (And nightly builds of thunderbird too) via their own ppa, called firefox-trunk https://launchpad.net/~ubuntu-mozilla-daily/+archive/ubuntu/ppa/+packages

Problem, they seem to also be called firefox-trunk internally, causing it to create their folders as ¨firefox-trunk¨ instead of the firefox-nightly. Obviously, just a little modification into the installation script could resolve that issue, and you can still install it manually of course.

_Oh, and Firefox-beta on their Debian repos, replace the default Firefox releases. Don’t ask me why, I would wish to stay on Arch if I could._

**Comment by manilarome** [comment]
Interesting... Thanks for the info! But sorry, I can't really fix this right now. I don't have a debian-based distros to test on. PRs are welcome of course!

**Comment by PoorPocketsMcNewHold** [comment]
By ¨fix¨. I would just suggest in your install script, when specified to install it on nightly, to check for a firefox-nightly folder first, and if not, search for a firefox-trunk folder, before giving up.

It’s just that. Nothing else really.

**Comment by manilarome** [comment]
Yes, you are right. The problem is how can we test it? I don't want to push
something that I know isn't tried and tested. If only Arch distributes its
firefox/mozilla programs like the one with debian/-based. So yeah, right
now, I can't really "fix" it because I cannot test it. PRs welcome! :)
